ui/src/tool-panel/track-number/track-number-infobox.tsx

```diff
@@ -169,19 +168,21 @@ const TrackNumberInfobox: React.FC<TrackNumberInfoboxProps> = ({
                             label={t('tool-panel.reference-line.end-location')}
                             value={<TrackMeter value={startAndEndPoints?.end?.address} />}
                         />
-                        {linkingState === undefined && referenceLine && userHasWriteRole && (
+                        {linkingState === undefined && referenceLine && (
                             <InfoboxButtons>
-                                <Button
-                                    variant={ButtonVariant.SECONDARY}
-                                    size={ButtonSize.SMALL}
-                                    onClick={() => {
-                                        getReferenceLineSegmentEnds(
-                                            referenceLine.id,
-                                            publishType,
-                                        ).then(onStartReferenceLineGeometryChange);
-                                    }}>
-                                    {t('tool-panel.location-track.modify-start-or-end')}
-                                </Button>
+                                <WriteRoleRequired>
```

Comment:
Tää kannattaa ehkä laittaa niin päin, että WriteRolen sisällä on InfoboxButtons koska InfoboxButtonsin mukana tulee joitain tyylityksiä

Reply:
No niinpäs tuleekin, korjattu